nilearn/plotting/cm.py

```diff
@@ -110,48 +102,47 @@ def alpha_cmap(color, name='', alpha_min=0.5, alpha_max=1.):
     red, green, blue = _colors.colorConverter.to_rgb(color)
     if name == '' and hasattr(color, 'startswith'):
         name = color
-    cmapspec = [(red, green, blue, 1.),
-                (red, green, blue, 1.),
-               ]
+    cmapspec = [(red, green, blue, 1.), (red, green, blue, 1.)]
     cmap = _colors.LinearSegmentedColormap.from_list(
-        '%s_transparent' % name, cmapspec, _rcParams['image.lut'])
+        f'{name}_transparent', cmapspec, _rcParams['image.lut'])
     cmap._init()
     cmap._lut[:, -1] = _np.linspace(alpha_min, alpha_max, cmap._lut.shape[0])
     cmap._lut[-1, -1] = 0
     return cmap
 
 
-
-################################################################################
+###############################################################################
```

Comment:
I would say that we either have them everywhere and we agree on a typical way to organize each module or we have them nowhere.

Since the latter is the norm in the codebase, I will remove them but I would not be against a soft guidelines to help organize function / classes in each file.

Reply:
OK I could not get myself to remote those headings as they do help read / navigate the code.